Aptoide/aptoide-client-v8
# [ASV-1771] Support themes in reactions button

**What does this PR do?**

   This PR aims at setting the theme in the reactions buttons. If the device has darktheme, then it should show the reaction button in white. Reverse should be black (like we have today).

**Database changed?**

No

**Where should the reviewer start?**

- [ ] EdtiroailBundleViewHolder.java

**How should this be manually tested?**

Open the app, in both themes (dark and light) and check if everything is correct regarding the reactions button in the different views (home, editorial detail, editorial bottom navigation section, etc)

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1771](https://aptoide.atlassian.net/browse/ASV-1771)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 
No.


**Code Review Checklist**
